core-services/importExport/src/main/java/org/silverpeas/core/importexport/versioning/VersioningImport.java

```diff
@@ -154,10 +135,9 @@ private SimpleDocument isDocumentExist(List<SimpleDocument> documents, Attachmen
   }
 
   public List<SimpleDocument> importDocuments(ForeignPK objectPK, List<Document> documents,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "importDocuments" is 22 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)